opentech/apply/funds/templates/funds/includes/review_table.html

```diff
@@ -1,33 +1,56 @@
 {% load review_tags %}
 <table class="reviews-sidebar js-reviews-sidebar">
-    {% if staff_reviews or object.staff_not_reviewed %}
-        {% if staff_reviews %}
-            <tr class="tr tr--subchild light-grey-bg">
-                <th colspan="2"></th>
-                <th>{{ object.reviews.submitted.recommendation|traffic_light }}</th>
-                <th>{{ object.reviews.submitted.get_score_display|default_if_none:'-' }}</th>
-            </tr>
-            {% include 'funds/includes/review_table_row.html' with reviews=staff_reviews %}
-        {% endif %}
+    {% if top_reviews %}
+        <tr class="tr tr--subchild light-grey-bg">
+            <th colspan="2"></th>
+            <th>{{ recommendation|traffic_light }}</th>
+            <th></th>
+        </tr>
+    {% endif %}
 
-        {% if object.staff_not_reviewed %}
-            {% include 'funds/includes/review_table_row.html' with reviews=object.staff_not_reviewed missing=True %}
-        {% endif %}
-    {% else %}
+    {% if reviews_block.role_reviewed or reviews_block.role_not_reviewed %}
+        {% for review_data in reviews_block.role_reviewed %}
+            {% include 'funds/includes/review_table_row.html' with review=review_data.review reviewer=review_data.reviewer role=review_data.role %}
+        {% endfor %}
```

Comment:
As previously mentioned the ordering is incorrect. Submitted reviews get higher priority over purely being assigned. 
```
{% for review_data in reviews_block.role_reviewed %}
...
{% for review_data in reviews_block.staff_reviewed %}
...
{% for review_data in reviews_block.role_not_reviewed %}
...
{% for review_data in reviews_block.staff_not_reviewed %}
```

Reply:
Got it sorry for missing this earlier